spec/models/year_statistic_spec.rb

```diff
@@ -1,6 +1,30 @@
-require "spec_helper.rb"
+# frozen_string_literal: true
+
+require "spec_helper"
 
 describe YearStatistic do
   it { is_expected.to belong_to :user }
   it { is_expected.to belong_to :site }
+
+  describe ".publications" do
+    it "should match the YearStatistic.publications method" do
+      VCR.use_cassette( "search_gbif_literature" ) do
+        publications = YearStatistic.publications( 2022 )
+        old_publications = YearStatistic.publications_old( 2022 )
+
+        publications[:results].each.with_index do | result, i |
+          expect( result.except( "_gbifDOIs", "title" ) ).to eq(
+            old_publications[:results][i].except( "_gbifDOIs", "title" )
+          )
```

Comment:
the official and unofficial apis returned slightly different titles. for example:
```
-"title" => "Updating the distribution of Vampyrum spectrum (Chiroptera, Phyllostomidae) in Colombia: new localities, potential distribution and notes on its conse…",
+"title" => "Updating the distribution of Vampyrum spectrum (Chiroptera, Phyllostomidae) in Colombia: new localities, potential distribution and notes on its conservation",
```